cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ListenersUtils.java

```diff
@@ -780,4 +780,18 @@ public static List<String> brokerExternalIPs(GenericKafkaListener listener, int
                     .findAny()
                     .orElse(null) : null;
     }
+
+    /**
+     * Returns whether to allocate NodePorts for LoadBalancer Service type
+     *
+     * @param listener Listener for which to allocate NodePorts
+     * @return         Whether to allocate NodePorts for Service
+     */
+    public static Boolean allocateLoadBalancerNodePorts(GenericKafkaListener listener) {
+        if (listener.getConfiguration() != null) {
+            return listener.getConfiguration().getAllocateLoadBalancerNodePorts();
+        } else {
+            return true;
+        }
```

Comment:
why not just ....

```java
return listener.getConfiguration() != null ? listener.getConfiguration().getAllocateLoadBalancerNodePorts() : true;
```

Reply:
No particular reason. I just copied what the other methods were doing. I was going to return null, so the Utils method takes over with the null check. But one of the bug check CI pipelines caught it as a bug.

Reply:
I think @ppatierno highlights what I missed ... it should probably return `null` (not configured) and not `true`. Whether it is in the longer code or shorter does not matter that much for me.

Reply:
Just ignored it in spotbugs. Hope that's alright

Reply:
Yeah, that is fine.